__tests__/app.test.js

```diff
@@ -49,6 +49,30 @@ describe("GET /api/topics", () => {
     });
 })
 
+
+describe("GET /api/articles", () => {
+    test("200 sends an array of articles to the client", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles).toBeSortedBy("created_at", {
+            descending: true,
+          });
+          expect(body.articles.length).toBe(13);
+          body.articles.forEach((article) => {
+            expect(typeof article.title).toBe("string");
+            expect(typeof article.topic).toBe("string");
+            expect(typeof article.author).toBe("string");
+            expect(typeof article.created_at).toBe("string");
+            expect(typeof article.article_id).toBe("number");
+            expect(typeof article.article_img_url).toBe("string");
+            expect(typeof article.comment_count).toBe("string");
+          });
+        });
+    });
+})
```

Comment:
Nice. All you need to add is a test that checks the articles have NOT got a body 👍